lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps a certain token amount for a minimum of another token amount.
```

Comment:
Swaps a certain token amount with fix amount value on input and minimum value amount on output.